addons/binding/org.openhab.binding.nest/ESH-INF/thing/thermostat.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="nest" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="thermostat" listed="false">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+
+		<label>Nest Thermostat</label>
+		<description>Provides the nest thermostat layer to talk to nest</description>
+
+		<channels>
+			<channel id="temperature" typeId="temperature" />
+			<channel id="humidity" typeId="humidity" />
+			<channel id="mode" typeId="mode" />
+			<channel id="previous_mode" typeId="mode">
+				<label>Previous Mode</label>
+				<description>The previous mode the thermostat was in</description>
+			</channel>
+			<channel id="max_set_point" typeId="setpoint">
+				<label>Max set point</label>
+				<description>Max set point in celcius</description>
+			</channel>
+			<channel id="min_set_point" typeId="setpoint">
+				<label>Min set point</label>
+				<description>Min set point in celcius</description>
+			</channel>
+			<channel id="can_heat" typeId="readonlyswitch">
```

Comment:
This is a static information and thus should rather be added as a property and not as a channel.